#! /usr/bin/env python

# Mutual camera localization configuration file

PACKAGE='dot_finder'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name             Type      Level Description             Default Min   Max
# Orange


gen.add("thresh_low", int_t,   0,    "thresh_low", 70, 0, 255)
gen.add("thresh_high", int_t,   0,    "thresh_high", 255, 0, 255)
gen.add("dilatation_size", double_t,   0,    "dilatation_size", 20, 0, 100)
gen.add("erosion_size", double_t,   0,    "erosion_size", 10, 0, 100)

exit(gen.generate(PACKAGE, "dot_finder", "DotFinder"))
